vxquery-core/src/main/java/org/apache/vxquery/xmlquery/query/XMLQueryCompiler.java

```diff
@@ -245,8 +240,6 @@ public Module getModule() {
                 RewriteRuleset.buildNestedDataSourceRuleCollection()));
         defaultLogicalRewrites.add(new Pair<AbstractRuleController, List<IAlgebraicRewriteRule>>(seqOnceCtrl,
                 RewriteRuleset.buildTypeInferenceRuleCollection()));
-        defaultLogicalRewrites.add(new Pair<AbstractRuleController, List<IAlgebraicRewriteRule>>(seqCtrlNoDfs,
```

Comment:
Do not remove this from master. This change is needed by the inclusion of new rewrite rules.